app/chitter.rb

```diff
@@ -0,0 +1,68 @@
+require 'sinatra/base'
+require_relative 'data_mapper_setup'
+require_relative 'helpers/application_helper'
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set :session_secret, '8h2e9h%$@##&#@&$)^@#ADIZdfgwd'
+
+  use Rack::MethodOverride
+
+  get '/' do
+    redirect to('/peeps')
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all.reverse
+    erb :'peeps/index'
+  end
+
+  post '/peeps' do
+    user = current_user
+    peep = user.peeps.create(content: params[:new_peep])
+    redirect to('/peeps')
+  end
+
+  get '/users/new' do
+    @user = User.new  # nb: this is to store the values in the form in case of unsuccessful registration
```

Comment:
Note your comment here is incorrect - this `@user` is only used to initialize the form the first time.  It can never include any values as this route is not posted to.